src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -168,5 +168,59 @@ public IDataView FilterByMissingValues(IDataView input, params string[] columns)
 
             return new NAFilter(Environment, input, complement: false, columns);
         }
+
+        /// <summary>
+        /// Skip <paramref name="count"/> rows in <paramref name="input"/>.
+        /// </summary>
+        /// <remarks>
+        /// Skips the first <paramref name="count"/> rows from <paramref name="input"/> and returns an <see cref="IDataView"/> with all other rows.
+        /// </remarks>
+        /// <param name="input">The input data.</param>
+        /// <param name="count">Number of rows to skip.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Skip](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/Skip.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public IDataView Skip(IDataView input, long count = SkipTakeFilter.Arguments.DefaultSkip)
```

Comment:
>SkipTakeFilter.Arguments.DefaultSkip [](start = 60, length = 36)

I'm somewhat on a fence regarding defaults for skip and take.
It feels weird to call skip without specifying amount.
Linq version doesn't have default value  for them as well. [https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.skip?view=netframework-4.7.2](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.skip?view=netframework-4.7.2)

Reply:
You are right. It doesn't make sense to have a default for Skip and Take. I'm removing and making sure the value is > 0.

---
In reply to: [254407426](https://github.com/dotnet/machinelearning/pull/2436#discussion_r254407426) [](ancestors = 254407426)